### PR TITLE
fix: add Sarah Lean

### PR DESCRIPTION
**Pull request by weeyin83**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/876"></a>

### CODE REVIEW

File: public/data/sarahlean.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      "url": "https://twitter.com/techielsas",
+      "icon": "twitter"
+    },
+    {
+      "name": "Learn more about Microsoft Azure on my YouTube channel",
+      "url": "https://youtube.com/techielass",
+      "icon": "youtube"
+    },
+    {
+      "name": "Follow me on LinkedIn",
```
Comment (Panquesito7) [inline public/data/sarahlean.json:23]:
```suggestion
      "name": "Connect with me on LinkedIn",
```

Comment (krishguptadev) [reply public/data/sarahlean.json:23]: This is personal choice of the author 😅 Please mark this as resolved.

File: public/data/sarahlean.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Sarah lean",
```
Comment (syedsohan) [inline public/data/sarahlean.json:2]:
```suggestion
  "name": "Sarah Lean",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/sarahlean.json (hunk 1 above)
Comment (weeyin83) [reply public/data/sarahlean.json:23]: Thanks for the suggestion @Panquesito7 but as @krishguptadev says, this is my personal choice.

**Review by syedsohan — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**